letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -560,7 +560,7 @@ BootstrapMageiaCommon() {
       exit 1
   fi
 
-  if ! $SUDO urpmi --force \
+  if ! $SUDO urpmi --force $QUIET_FLAG \
```

Comment:
You still can if you want so you get credit for it in our git history, but I don't think it's a big deal either way.